austation/code/modules/projectiles/guns/misc/chem_gun.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/gun/chem/medical
+	name = "medical darter"

```

Comment:
Dart gun would sound better

Reply:
Dart pistol exists, and having a dart gun and dart pistol is going to confuse people on comms.

Reply:
I don't think there's such thing as a medical dart gun, unless you are referring to the syringe gun